test/XRP/issued-currency-client.test.ts

```diff
@@ -71,6 +71,25 @@ describe('Issued Currency Client', function (): void {
     })
   })
 
+  it('getTrustLines - invalid peerAccount', function (done): void {
+    // GIVEN an IssuedCurrencyClient
+    const issuedCurrencyClient = new IssuedCurrencyClient(
+      fakeSucceedingGrpcClient,
+      fakeSucceedingJsonClient,
+      XrplNetwork.Test,
+    )
+
+    // WHEN getTrustLines is called with an invalid peerAccount address THEN an error is propagated.
+    const peerAddress = 'malformedAddress'
+    issuedCurrencyClient
+      .getTrustLines(testAddress, peerAddress)
+      .catch((error) => {
+        assert.typeOf(error, 'Error')
+        assert.equal(error, XrpError.xAddressRequired)
+        done()
+      })
```

Comment:
can we just await `issuedCurrencyClient.getTrustLines` and use a `try/catch` instead of a promise? you don't need the `done` callback anymore bc it acts synchronously

i noticed today that we do this in a lot of places in the sdk but i think it's confusing...it's the same thing as doing

```
try {
  await myFunction()
} catch (error) {
  errorHandler()
}
done()
```

note that you need to return the `done` callback from the test call or else it will yell at you